server/src/controllers/questionsController.ts

```diff
@@ -313,3 +314,29 @@ export const createCommentHandler = async (req: Request, res: Response) => {
       .json({ error: "Error Adding Your Comment to the Database" });
   }
 };
+
+export const editQuestionHandler = async (req: Request, res: Response) => {
+  const questionId = Number(req.params.id);
+
+  if (!questionId) {
+    res.status(400).json({ error: "Question Id was not provided" });
+  }
+
+  const question = req.body.questionContent;
+
+  if (!question) {
+    res.status(400).json({ error: "Question was not provided" });
+  }
+
+  try {
+    const updateQuestionQuery = await editQuestion(questionId, question);
+    logger.info({
+      message: "updateQuestionQuery data",
+      value: updateQuestionQuery
+    });
+    const data = updateQuestionQuery[0];
```

Comment:
```suggestion

    const data = updateQuestionQuery[0];
    logger.info({  
      message: "editQuestionHandler data",  
      value: data  
    }); 

```
add spaces and logger